python_ta/__init__.py

```diff
@@ -358,13 +368,17 @@ def get_file_paths(rel_path: AnyStr) -> Generator[AnyStr, None, None]:
                 yield os.path.join(root, filename)  # Format path, from root.
 
 
-def _verify_pre_check(filepath: AnyStr) -> bool:
-    """Check student code for certain issues."""
+def _verify_pre_check(filepath: AnyStr, allow_pylint_comments: bool) -> bool:
+    """Check student code for certain issues.
+    The additional allow_pylint_comments parameter indicates whether we want the user to be able to add comments
+    beginning with pylint which can be used to locally disable checks."""
```

Comment:
Put the ending `"""` on a new line (this is the style we use when there's a multi-line docstring)